### DISCUSSION TITLE
Updated to Android 15 and sound stopped working

### DISCUSSION THREAD

**Discussion by henkiewie** [termux/termux-packages#22698] [Q&A]
HI,

I asked the same question, but its maybe in the wrong section, so I will try again here.

I was happily using pulseaudio in combination with squeezelite on my oneplus tablet.
The other day I updated to the release candidate of Android 15.
But for now there is no sounddevice anymore.
Even pulseaudio is not started anymore, but thats maybe because of that.
I can start it by hand pulseaudio --start, but nothing changed.
I reinstalled termux (f-droid) and allowed external apps.
Pacmd --list-cards gives zero cards.

Is there anyway to fix this? I only have a dummy sound device available.
I'm using the oneplus pad. It worked on android 14 and its not easy to go back.

Any help is appreciated.

**Comment by tmakarios** [comment]
A different but possibly related experience:

I was using a first-generation Pixel XL as a remotely controlled jukebox via VLC on Termux on DivestOS 18.1, and it was working fine.  But now DivestOS is no longer being maintained, I tried upgrading a first-generation Pixel to LineageOS 22.1 (based on Android 15), to replace it.

The upgrade mostly went fine, but using it as a jukebox isn't a great experience.  Although there is audio output via PulseAudio, it keeps stuttering.  I've tried hugely increasing `network-caching` in `vlcrc`, and `default-fragments` and `default-fragment-size-msec` in `usr/etc/pulse/daemon.conf`, and even fiddling with other settings, to avoid resampling entirely, but none of it solved the issue.  Playing via the sox `play` command reduces the stuttering, but doesn't eliminate it.

The Pixel XL with DivestOS 18.1 (based on Android 11, or thereabouts) still does the job without any stuttering, but I'm reluctant to keep using it with an unsupported operating system for any length of time, even only as a jukebox.

**Reply by Biswa96** [reply]
Any kind of logs and steps to reproduce it may help to troubleshoot the issue.

**Reply by tmakarios** [reply]
Steps to reproduce:

1. Install LineageOS 22.1 on a first-generation Google Pixel.  Maybe other phones would exhibit the bug, too, maybe other variants of Android 15 would exhibit variants of the bug, as @henkiewie's experience suggests; I don't know.
2. Install Termux on the phone.
3. In Termux, run:
```
pkg install vlc
vlc http://radionz-ice.streamguys.com/concert.mp3
```
4. Listen to the stuttering audio.

I'm not aware of any relevant logs.  Did you have anything in particular you wanted to see?
